README.md

```diff
@@ -150,7 +150,7 @@ each milestone.
 
 ## nbconvert
 
-### Next
+### Next (4.2)
```

Comment:
Doesn't it make more sense to only label it `### Version 4.2`? That way you don't need to update it every time you do an update in one of these packages...?